### PR TITLE
Add contour conformance report for v1.0.0

### PR DESCRIPTION
**Pull request by sunjayBhatia**
**What type of PR is this?**

/area conformance

**What this PR does / why we need it**:

Contour v1.28.1 supports Gateway API v1.0.0, add conformance report for this version.

**Which issue(s) this PR fixes**:

N/A

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by skriss — APPROVED** [review]

**Comment by robscott** [comment]
Thanks @sunjayBhatia! This LGTM, but I'm not sure where we are in the transition between report structures here.

@mlavacca @shaneutt Are we holding off on using the new reporting structure until the v1.1 release?

File: conformance/reports/v1.0.0/projectcontour-contour.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+apiVersion: gateway.networking.k8s.io/v1alpha1
```
Comment (robscott) [inline conformance/reports/v1.0.0/projectcontour-contour.yaml:1]: Recommend updating implementations page to link to this report as well (https://gateway-api.sigs.k8s.io/implementations/#contour)

**Review by robscott — COMMENTED** [review]
I think both @shaneutt and @mlavacca are OOO, and since no other reports have migrated to the new structure yet, I think it's fine to merge this as is. Just have one non-blocking suggestion.

/approve

File: conformance/reports/v1.0.0/projectcontour-contour.yaml (hunk 1 above)
Comment (sunjayBhatia) [reply conformance/reports/v1.0.0/projectcontour-contour.yaml:1]: good catch, done!

**Pull request merged**